airflow-core/src/airflow/api_fastapi/common/parameters.py

```diff
@@ -868,6 +868,15 @@ def _transform_ti_states(states: list[str] | None) -> list[TaskInstanceState | N
         filter_param_factory(TaskInstance.queue, list[str], FilterOptionEnum.ANY_EQUAL, default_factory=list)
     ),
 ]
+QueryTIPoolPatternSearch = Annotated[
+    _SearchParam,
+    Depends(search_param_factory(TaskInstance.pool, "pool_pattern")),
```

Comment:
```suggestion
    Depends(search_param_factory(TaskInstance.pool, "pool_name_pattern")),
```


Since we have already used `pool_name_pattern` for other endpoint, would it be better to use `pool_name_pattern` instead of `pool_pattern` ?
https://github.com/apache/airflow/blob/16c544079dab9cd5967951b4dc8076ade19f0ca8/airflow-core/src/airflow/api_fastapi/common/parameters.py#L964-L96

Reply:
Thanks for reviewing!
I’ve updated the parameter to use `pool_name_pattern`